docs/releases/5.1.txt

```diff
@@ -78,10 +78,12 @@ Minor features
 * The new :meth:`.OGRGeometry.set_3d` method allows addition and removal of the
   ``Z`` coordinate dimension.
 
-* :class:`~django.contrib.gis.gdal.OGRGeometry` and
-  :class:`~django.contrib.gis.gdal.Point` now support measured geometries
-  via the new :attr:`.OGRGeometry.is_measured` and :attr:`.Point.m` properties,
-  and the :meth:`.OGRGeometry.set_measured` method.
+* :class:`~django.contrib.gis.gdal.OGRGeometry`,
+  :class:`~django.contrib.gis.gdal.Point` and
+  :class:`~django.contrib.gis.gdal.LineString` now support measured geometries
+  via the new :attr:`.OGRGeometry.is_measured`, :attr:`.Point.m` and
```

Comment:
```suggestion
  via the new :attr:`.OGRGeometry.is_measured`, :attr:`.Point.m`, and
```

Reply:
Ah yes, I forgot about the oxford comma preference. However, I'm not sure about this one. 

We've two groups, a list of properties and a method. Currently there's a comma before the 'and' that joins these two groups (line 85). Adding another one here makes the distinction between the two groups less clear (for me).

Reply:
I think it's clearer without references.